examples/electrophysiology/neuromuscular/cuboid_muscle_with_prestretch/src/muscle_with_prestretch.cpp

```diff
@@ -31,6 +31,45 @@ using FiberDiffusionSolver = TimeSteppingScheme::ImplicitEuler<T>;
 #define N_ALGEBRAICS 19
 #endif
 
+// define material
+struct Material : Equation::SolidMechanics::HyperelasticityBase {
+  static constexpr bool isIncompressible =
+      true; //< if the formulation is incompressible, then,
+            // strainEnergyDensityFunctionVolumetric will not be considered
+  static constexpr bool usesFiberDirection =
+      false; //< if the decoupled form uses the 4th or 5th invariants, Ibar4,
+             // Ibar2, this means it is an anisotropic material
+  static constexpr bool usesActiveStress =
+      false; //< if the value of an active stress term will be added to the
+             // stress
+
+  // material parameters
+  static constexpr auto c1 = PARAM(0); //< material parameter
+  static constexpr auto c2 = PARAM(1); //< material parameter
+
+  static constexpr int nMaterialParameters =
+      2; //< number of material parameters
+
+  //! the isochoric part of the decoupled strain energy function,
+  //! Ψ_iso(Ibar1,Ibar2,Ibar4,Ibar5), in terms of the reduced invariants
+  static const auto constexpr strainEnergyDensityFunctionIsochoric =
```

Comment:
For variable declarations, `constexpr` implies `const`, so having both modifiers sounds a bit strange (but totally legal).

Read also https://stackoverflow.com/a/14117121/2254346